File: 30 days of code( python)/Day 18.py
class Solution:
    def __init__(self):
        self.my_stack = []
        self.my_queue = []
        
    # Write your code here
    def pushCharacter(self, ch):
        self.my_stack.append(ch)
        
    
    def enqueueCharacter(self, ch):
        self.my_queue.insert(0, ch) # 0 is for the position!
        
    
    def popCharacter(self):
        return self.my_stack.pop()
    
    def dequeueCharacter(self):
        return self.my_queue.pop()

File: 30daysOFcode_JavaScript/day 18.js
function Solution() {
  //Write your code here
  this.stack = [];
  this.queue = [];

  this.pushCharacter = this.stack.push;
  this.popCharacter = this.stack.pop;
  this.enqueueCharacter = this.queue.push;
  this.dequeueCharacter = this.queue.shift;
}
